applications/dashboard/models/class.activitymodel.php

```diff
@@ -876,7 +876,11 @@ public function sendNotification($ActivityID, $Story = '', $Force = false) {
                     ->setTitle($ActivityHeadline);
 
                 if ($message = val('Story', $Activity)) {
-                    $emailTemplate->setMessage($message, true);
+                    $prefix = c('Garden.Email.Prefix', '');
+                    if (!empty($prefix)) {
```

Comment:
I'd say let's not do this ghost-break thing unless it breaks the world without it. If you want to prefix and you want it broken from the rest of the content, add breaks too to the prefix.